src/qt/diagnosticsdialog.cpp

```diff
@@ -262,765 +293,36 @@ void DiagnosticsDialog::on_testButton_clicked()
-#else
-    connect(m_udpSocket, &QUdpSocket::errorOccurred, this, &DiagnosticsDialog::clkSocketError);
-#endif
-
-    if (!NTPHost.addresses().empty())
-    {
-        m_udpSocket->connectToHost(QHostAddress(NTPHost.addresses().first()), 123, QIODevice::ReadWrite);
-    }
-    else
-    {
-        clkSocketError();
-    }
-}
-
-void DiagnosticsDialog::clkStateChanged(QAbstractSocket::SocketState state)
-{
-    if (state == QAbstractSocket::ConnectedState)
-    {
-        connect(m_udpSocket, &QUdpSocket::readyRead, this, &DiagnosticsDialog::clkFinished);
-
-        char NTPMessage[48] = {0x1b, 0, 0, 0 ,0, 0, 0, 0, 0};
-
-        m_udpSocket->write(NTPMessage, sizeof(NTPMessage));
-    }
-}
-
-void DiagnosticsDialog::clkSocketError()
-{
-    m_udpSocket->close();
-
-    clkReportResults(0, true);
-}
-
-void DiagnosticsDialog::clkFinished()
-{
-    if (m_udpSocket->waitForReadyRead(10 * 1000))
-    {
-        int64_t start_time = GetAdjustedTime();
-
-        // Only allow this loop to run for 5 seconds maximum.
-        while (m_udpSocket->hasPendingDatagrams() && GetAdjustedTime() - start_time <= 5)
-        {
-            QByteArray BufferSocket = m_udpSocket->readAll();
-
-            if (BufferSocket.size() == 48)
-            {
-                int nNTPCount = 40;
-                uint32_t DateTimeIn = uchar(BufferSocket.at(nNTPCount))
-                        + (uchar(BufferSocket.at(nNTPCount + 1)) << 8)
-                        + (uchar(BufferSocket.at(nNTPCount + 2)) << 16)
-                        + (uchar(BufferSocket.at(nNTPCount + 3)) << 24);
-                time_t tmit = ntohl(DateTimeIn) - 2208988800U;
-
-                m_udpSocket->close();
-
-                boost::posix_time::ptime localTime = boost::posix_time::microsec_clock::universal_time();
-                boost::posix_time::ptime networkTime = boost::posix_time::from_time_t(tmit);
-                boost::posix_time::time_duration timeDiff = networkTime - localTime;
-
-                clkReportResults(timeDiff.total_seconds());
-
-                return;
-            }
-        }
-    }
-    else // The other state here is a socket or other indeterminate error such as a timeout (coming from clkSocketError).
-    {
-        // This is needed to "cancel" the timeout timer. Essentially if the test was marked completed via the normal exits
-        // above, then when the timer calls clkFinished again, it will hit this conditional and be a no-op.
-        if (GetTestStatus("VerifyClock") != completed)
-        {
-            clkReportResults(0, true);
-        }
-
-        return;
-    }
-}
-
-void DiagnosticsDialog::clkReportResults(const int64_t& time_offset, const bool& timeout_during_check)
-{
-    if (!timeout_during_check)
-    {
-        if (abs64(time_offset) < 3 * 60)
-        {
-            UpdateTestStatus("VerifyClock", ui->verifyClockResultLabel, completed, passed);
+    DiagnoseLib::Diagnose::setResearcherModel();
+
+    for(auto &i: m_diagnostic_tests){
```

Comment:
This is failing the linter. The code style requires

for (.....) {
}

There must be a space between the for keyword and the parenthesis, and also a space between the parenthesis and the opening brace.

Reply:
I reran the linter and fixed the issues. But this one did not come out when running the linter.